packages/backend/src/accounting/psql/ledger-transfer/index.ts

```diff
@@ -30,6 +31,7 @@ interface ValidateTransferArgs {
   timeoutMs?: bigint
   creditAccount: LedgerAccount
   debitAccount: LedgerAccount
+  transferRef: string
```

Comment:
At this point, `ValidateTransferArgs` could be `Omit<CreateTransferArgs, 'type'>` or just use `args: CreateTransferArgs` in `validateTransfer`